src/projectLogic.js

```diff
@@ -0,0 +1,104 @@
+/* global document, localStorage */
+
+import project from "./project";
+import todoItemLogic from "./todoItemLogic";
+
+
+const projectLogic = (() => {
+    const getData = () => {
+        return JSON.parse(localStorage.getItem('projectsArr') || "[]");
```

Comment:
The initial `projectsArr` when checked in `localStorage` doesn't contain `Default` project. I suppose since we need to have this `Default` project our localStorage['projectsArr'] should always have that `Default` project no matter what.

Reply:
@rvvergara, we solved the problem differently, since `Default` will always be static, we felt no need to add it to the array.